src/Security/Authentication/SamlAuthenticator.php

```diff
@@ -52,22 +54,35 @@ class SamlAuthenticator extends AbstractAuthenticator implements InteractiveAuth
      * @SuppressWarnings(PHPMD.ExcessiveParameterList)
      */
     public function __construct(
-        private IdentityProvider $identityProvider,
-        private ServiceProvider $serviceProvider,
-        private RedirectBinding $redirectBinding,
-        private SamlAuthenticationStateHandler $samlAuthenticationStateHandler,
-        private ProcessSamlAuthenticationHandler $processSamlAuthenticationHandler,
-        private AuthenticationSuccessHandlerInterface $successHandler,
-        private AuthenticationFailureHandlerInterface $failureHandler,
-        private SamlProviderInterface $samlProvider,
-        private RouterInterface $router,
-        private LoggerInterface $logger,
-        private string $acsRouteName
+        private readonly IdentityProvider $identityProvider,
+        private readonly ServiceProvider $serviceProvider,
+        private readonly RedirectBinding $redirectBinding,
+        private readonly SamlAuthenticationStateHandler $samlAuthenticationStateHandler,
+        private readonly ProcessSamlAuthenticationHandler $processSamlAuthenticationHandler,
+        private readonly AuthenticationSuccessHandlerInterface $successHandler,
+        private readonly AuthenticationFailureHandlerInterface $failureHandler,
+        private readonly SamlProviderInterface $samlProvider,
+        private readonly RouterInterface $router,
+        private readonly LoggerInterface $logger,
+        private readonly string $acsRouteName,
+        private readonly BridgeContainer $bridgeContainer //Todo paul: Dit moet niet zo werken. zie comment hieronder
     ) {
     }
 
+    /**
+     * Todo Paul: in services_authentication.yaml::Surfnet\SamlBundle\Security\Authentication\SamlAuthenticator
```

Comment:
TODO Paul?

Reply:
Yup, during one of my 'Saving private Paul' rescue missions I left you this todo mission. Granted, it is quite some time ago. Nevertheless we need to address this. 

Do you feel up to the task?